### DISCUSSION TITLE
Reconfigure logo on Form view [Oct 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4117] [Discord Threads]
> 🙋‍♂️ **emdax**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not supported  
#  
> 🙋‍♂️ **emdax**  
#### not supported?  
#  
> 🙋‍♂️ **emdax**  
#### like you don't support that option?  
#  
> 👨‍🦲 **dstala**  
#### Yes. Currently custom logo in forms is not supported.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1025509491762012232)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
